plugins/modules/aci_dhcp_relay.py

```diff
@@ -94,6 +126,16 @@
     state: query
   delegate_to: localhost
   register: query_result
+
+- name: Query all DHCP relay policies
```

Comment:
Is there a way to query all infra DHCP relay policies? We may need to change the description of the tenant above (When tenant is not provided the module will be applied to the global (infra) policy.)

Reply:
I don't think so since we cannot filter on infra since it is not separated as a class in url construction. So when we do not provide anything it will be class query for relay policies thus displaying all of a class.  A user could still filter out the response to only handle infra objects.

Other option which I think is possible is to provide infra as root_class. The downside would then be that you will not be able to display all relay policies, but you would show infra with it children. So you would have a nested response. Think this deviates from the behaviour we currently have.